closure/goog/net/jsloader.js

```diff
@@ -111,8 +123,10 @@ goog.net.jsloader.loadMany = function(uris, opt_options) {
   if (isAnotherModuleLoading) {
     // jsloader is still loading some other scripts.
     // In order to prevent the race condition noted above, we just add
-    // these URIs to the end of the scripts' queue and return.
-    return;
+    // these URIs to the end of the scripts' queue and return the deferred
```

Comment:
done. not sure what capitalization refers to. deferred? have changed sentence to work around.